CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/whitaker-io/machine/pullRequest?prid=6375353)